app/services/setting_registry.rb

```diff
@@ -58,7 +58,7 @@ def [](name)
       logger.warn("Setting #{name} has no definition, please define it before using") if ready?
       return
     end
-    definition.value
+    definition.value || definition.default
```

Comment:
If `definition.value` is `false` it'll also use `definition.default`. This must check for `nil?`.

Reply:
if I would read comments sooner then debugging, I could have saved myself a little bit of struggle, you are correct and I've fixed it :+1: